app/views/contests/_admin.html.erb

```diff
@@ -16,3 +16,7 @@
 <% end %>
 </ul>
 
+<p>
+  <b> Problem set: </b>
+  <%= link_to @contest.problem_set.name, @contest.problem_set %>
```

Comment:
How about moving the problem set link to the top of the "problems" tab? As written it takes up space on all tabs, including the scoreboard.

(Maybe in separate PR we could also move the existing groups UI into one of the tabs, I'm thinking either the "contestants" tab, or a new "groups" tab; if we use the "contestants" tab, my preferred UI is: list of groups with access, button to add a group, button to register a single user, then list of contestants)

Reply:
I'm fine with that.

Reply:
Done. While doing this, I noticed invalid `<p>` tags around the problems table, so I removed them (see commit df5c0536f4affe0af6830940c522c30fc1497f99). They had the effect of adding blank space above and below the problems table. I don't think it's intentional (the scoreboard and supervisor tables don't such space). Here is what it looks like before and after.

Before (with space above and below the problems table due to `<p>` tags):

![1-old](https://user-images.githubusercontent.com/5887562/108508553-dedb6500-7320-11eb-9602-553cbdb9c8a0.png)

After removing the `<p>` tags:

![2-new](https://user-images.githubusercontent.com/5887562/108508586-ea2e9080-7320-11eb-8738-6a49ade1420d.png)

After also moving the problem set link:

![3-new-moved-problem-set](https://user-images.githubusercontent.com/5887562/108508604-ef8bdb00-7320-11eb-8aaa-eae7fc345b38.png)

Reply:
Looks good to me.